aspnet/data/entity-framework-6.rst

```diff
@@ -0,0 +1,136 @@
+        {
+        }
+    }
+
+In the ``Startup`` class within ``ConfigureServices`` add factory method of your context with it's connection string. Context should be resolved once per scope to ensure performance and ensure reliable operation of Entity Framework. 
+
+.. code-block:: c#
+    :linenos:
+    
+    public void ConfigureServices(IServiceCollection services)
+    {
+        services.AddScoped((_) => new ApplicationDbContext(Configuration["Data:DefaultConnection:ConnectionString"]));
+        
+        // Configure remaining services
+    }
+
+Migrate configuration from config to code
+-----------------------------------------
+
+Entity Framework 6 allows configuration to be specified in xml (in web.config or app.config) or through code. As of ASP.NET 5, all configuration is code-based.
+
+Code-based configuration is achieved by creating a subclass of ``System.Data.Entity.Config.DbConfiguration`` and applying ``System.Data.Entity.DbConfigurationTypeAttribute`` to your ``DbContext`` subclass.
+
+Our config file typically looked like this:
+
+.. code-block:: xml
+    :linenos:
+    
+    <entityFramework>
+        <defaultConnectionFactory type="System.Data.Entity.Infrastructure.LocalDbConnectionFactory, EntityFramework">
+            <parameters>
+                <parameter value="mssqllocaldb" />
+            </parameters>
+        </defaultConnectionFactory>
+        <providers>
+            <provider invariantName="System.Data.SqlClient" type="System.Data.Entity.SqlServer.SqlProviderServices, EntityFramework.SqlServer" />
+        </providers>
+    </entityFramework>
+
+The ``defaultConnectionFactory`` element sets the factory for connections. If this attribute is not set then the default value is ``SqlConnectionProvider``. If, on the other hand, value is provided, the given class will be used to create ``DbConnection`` with its ``CreateConnection`` method. If the given factory has no default constructor then you must add parameters that are used to construct the object.
+
+.. code-block:: c#
+    :linenos:
+
+    [DbConfigurationType(typeof(CodeConfig))] // point to the class that inherit from DbConfiguration
+    public class ApplicationDbContext : DbContext
+    {
+        [...]
+    }
+    
+    public class CodeConfig : DbConfiguration
+    {
+        public CodeConfig()
+        {
+            SetDefaultConnectionFactory(new System.Data.Entity.Infrastructure.SqlConnectionFactory());
+            SetProviderServices("System.Data.SqlClient",
+                System.Data.Entity.SqlServer.SqlProviderServices.Instance);
+            }
+        }
+    }
+    
+SQL Server and SQL Server Express
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+This is the default and so no explicit configuration is needed. Optionally the above ``CodeConfig`` class can be used to explicitly set the connection factory and provider services.
+
+SQL Server Express LocalDB
+~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
Given you are passing in a connection string, you shouldn't really need this. This is only needed if you are passing a database name and want EF to use the connection factory to work out which server to use. I wouldn't push folks down this path, but rather just tell them to pass in the whole connection string (as you have shown so far in the article).